docs/guides/monitoring-mode.mdx

```diff
@@ -9,50 +9,63 @@ import { RunPkgCommand } from "@site/src/partials";
 
-In this mode, screenshots from your pull request branch are compared with screenshots from a [baseline build](/baseline-build) determined by Git history analysis and [other criterias](/baseline-build).
+During your CI pipeline, screenshots of the application are captured on each pull request and on the default branch (main, production, etc.). These screenshots are uploaded to Argos, where they are compared against a [baseline build](/baseline-build).
+
+The [baseline build](/baseline-build) represents the most recent approved state of your application and is automatically selected based on Git history analysis and [other criteria](/baseline-build).
+
+In the Argos app, you can review highlighted differences, approve intended updates, and block regressions before merging.
+
+**Note:** Your CI pipeline must also run on the default branch (main, production, etc.) to keep the baseline build up to date.
 
 ### Workflow
 
 1. **Feature Development**
 
-- A developer commits code to a feature or bugfix branch.
-- Optionally, a pull request (PR) is opened for review.
+   - A developer creates a feature (or bugfix) branch and commits changes.
+   - Optionally, a pull request (PR) is opened.
 
-2. **Visual Tests on the Feature Branch**
+2. **Visual Tests in CI**
 
-- Argos integrates with your test framework [via SDKs](/getting-started).
-- During CI, your test framework runs visual tests.
-- Screenshots are captured and uploaded to Argos.
+   - When the pull request is created, the CI pipeline runs automated tests.
+   - During these tests, your test framework (Playwright, Cypress, etc.) captures screenshots of the app. Argos offers [SDKs](/getting-started) for easy integration with popular frameworks.
+   - At the end of the tests, screenshots are uploaded to Argos automatically with the SDK, or manually using the Argos CLI.
 
-3. **Baseline Comparison**
+3. **Comparison with Baseline**
 
-- Argos determines the [baseline build](/baseline-build) from Git history and other criteria.
-- New screenshots are compared against the baseline.
+   - Argos receives the build containing screenshots and metadata.
+   - It automatically determines the [baseline build](/baseline-build) using Git history analysis and other criteria.
+   - The new screenshots are compared against the baseline.
 
 4. **Results & Notifications**
 
-- When the build completes:
-  - ✅ No differences → commit status set to _success_.
-  - ❌ Differences found → commit status set to _failed_, with diffs visible in the Argos dashboard.
-- On GitHub, Argos also posts a summary comment with a link to the dashboard.
+   - The build is complete once all screenshots have been compared:
+     - ✅ No differences → commit status set to _success_.
+     - ❌ Differences found → commit status set to _failed_. The changes are visible in the Argos app for review and approval.
+   - Argos notifies GitHub about the build status and posts a summary comment on the pull request.
```

Comment:
Keep the previous one, if they use GitHub we post a comment. It makes it like you can only use GitHub.